.github/actions/backend-deploy-job/files/cloudbuild.yaml

```diff
@@ -85,7 +85,7 @@ steps:
 
         # Extract VPC connector and environment variables
         export VPC_CONNECTOR=$(awk -F '=' '/^VPC_CONNECTOR/ {print $2}' "./devops/vaults.${env_name}")
-        export VAL=$(awk -F= '{gsub(/"/, "", $1); gsub(/"/, "", $2); printf "- name: %s\n  value: %s\n", $1, $2}' "./devops/vaults.${env_name}")
+        export VAL=$(awk '{f1=f2=$0; sub(/=.*/,"",f1); sub(/[^=]+=/,"",f2); printf "- name: %s\n  value: %s\n",f1,f2}' "./devops/vaults.${env_name}" | sed 's/"/"/g')
```

Comment:
what does this do: s/"/"/g  ---> is it just replacing " with " or am i wrong?

Reply:
maybe should be 's/"/\\"/g' ?

Reply:
github does not display double backslash... in my comment above

Reply:
should be "sed 's/\"/\"/g'" to escape double quotes, not sure the backslash disappear. I will verify it.